be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -108,9 +109,13 @@ size_t ScanOperator::_buffer_unplug_threshold() const {
 }
 
 bool ScanOperator::has_output() const {
-    if (_is_finished) {
+    if (_is_finished || (_is_epoch_finished && _is_stream)) {
```

Comment:
Can we override this method in `StreamScanOperator`?

Reply:
If override this method in `StreamScanOperator`, there will be much repetitive code